.github/workflows/testing.yml

```diff
@@ -75,7 +76,49 @@ jobs:
       - name: Test
         env:
           MP_API_KEY: ${{ secrets.MP_API_KEY }}
-        run: pytest --cov=atomate2 --cov-report=xml
+        run: pytest --ignore=tests/forcefields --cov=atomate2 --cov-report=xml
+
+      - uses: codecov/codecov-action@v1
+        if: matrix.python-version == '3.10' && github.repository == 'materialsproject/atomate2'
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          file: ./coverage.xml
+
+  test-forcefields:
+    services:
+      local_mongodb:
+        image: mongo:4.0
+        ports:
+          - 27017:27017
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[strict,strict-forcefields,tests]
+          pip install torch-runstats
+          pip install --no-deps nequip==0.5.6
+
+      - name: Install pymatgen from master if triggered by pymatgen repo dispatch
+        if: github.event_name == 'repository_dispatch' && github.event.action == 'pymatgen-ci-trigger'
+        run: pip install --upgrade 'git+https://github.com/materialsproject/pymatgen@${{ github.event.client_payload.pymatgen_ref }}'
+
+      - name: Test
+        env:
+          MP_API_KEY: ${{ secrets.MP_API_KEY }}
+        run: pytest --cov=atomate2 --cov-report=xml tests/forcefields
```

Comment:
should we merge the force field tests into the main test run? ideally, we'll use `pytest-split` later to split workload into multiple runners equally to speed up CI

Reply:
The way it's partitioned now is because `tblite` is incompatible with `torch`. The safest approach, to me, seemed like completely quarantining the `torch`-dependent modules (just `forcefields`) from everything else. Open to suggestions